src/js/background/background.js

```diff
@@ -280,39 +289,54 @@ async function displayBrowserActionBadge() {
   }
 }
 
-browser.runtime.onMessage.addListener(async (m) => {
+browser.runtime.onMessage.addListener(async (m, sender, sendResponse) => {
   let response;
+  let tab2id;
```

Comment:
question: Is this necessary? It looks like only assignt his value to `sender.tab.id` during `fillInputWithAliasParentPage`, but then we don't do anything with it?

Reply:
Removed in 04ab44e6c577e900707d24673a78229a1307502a